Overview.bs

```diff
@@ -91,22 +91,99 @@ Introduction {#intro}
 
 <em>This section is not normative.</em>
 
+Incremental Font Transfer (IFT) is a collection of technologies to improve the latency of remote fonts (or "web fonts") on the web. Without this technology, a browser needs to download every last byte of a font before it can render any characters using that font. IFT allows the browser to download only some of the bytes in the file, thereby decreasing the perceived latency between the time when a browser realizes it needs a font and when the necessary text can be rendered with that font.
+
 The success of WebFonts is unevenly distributed. This specification allows WebFonts to be used where
```

Comment:
> The success of WebFonts is unevenly distributed.

Should we elaborate on this a bit further? For someone who haven't read PFE Evaluation Report, this statement may not seem obvious or self-explanatory.